service/controller/v24/controllercontext/status.go

```diff
@@ -1,25 +1,22 @@
 package controllercontext
 
-type Status struct {
-	// Cluster carries information between cluster controller resource.
```

Comment:
The controller separation on this level does not work. We have the workerasgname resource run in all controllers.